comment on exception __EXCEPTION_NAME__ is null 